test/api/liskApi.js

```diff
@@ -272,14 +274,27 @@ describe('Lisk.api()', () => {
 
 		it('should list standby delegates', () => {
 			const callback = sinon.spy();
-			const options = { limit: '1', orderBy: 'rate:asc', offset: 101 };
+			const options = { limit: requestDataLimit, orderBy: 'rate:asc', offset: requestOffset };
```

Comment:
We should use `offset` and `orderBy` values which are different from the defaults in this test.